esp-hal/MIGRATING-0.20.md

```diff
@@ -159,3 +159,14 @@ You can pass `NoPin` or `Level` as inputs, and `NoPin` as output if you don't ne
 -    .with_pins(Some(sclk), Some(mosi), NO_PIN, NO_PIN);
 +    .with_pins(sclk, mosi, Level::Low, NoPin);
 ```
+
+### Placing drivers in RAM is now done via esp-config
+
+We've replaced some usage of features with [esp-config](https://docs.rs/esp-config). Please remove any reference to `place-spi-driver-in-ram` in your `Cargo.toml` and migrate to the `[env]` section of `.cargo/config.toml`.
+
+```diff
+# feature in Cargo.toml
+- esp-hal = { version = "0.20", features = ["place-spi-driver-in-ram"] }
```

Comment:
This looks like we can remove the whole line 🤔

Reply:
:shrug:, it's explained in words above, I think that should be enough.